include/zephyr/drivers/seu/seu.h

```diff
@@ -0,0 +1,410 @@
+/*
+ * Copyright (c) 2023 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_SEU_H_
+#define ZEPHYR_INCLUDE_SEU_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define SEU_MAX_FRAME 2
+
+/**
+ * Structure to hold user input to inject safe SEU error frame data.
+ */
+struct inject_safe_seu_error_frame {
+	/* Sector address */
```

Comment:
```suggestion
	/** Sector address */
```

Please check other similar comments elsewhere in the file. Note that there are also fields for which comments are missing altogether